src/modules/App.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import './App.css';
+import Buttons from './Buttons';
+import Upper from './Upper';
+
+// eslint-disable-next-line react/prefer-stateless-function
+export default class App extends React.Component {
```

Comment:
Great job making your App component class based, however kindly ensure it imports the Calculator.js component after you create the Calculator component